test/bdd/features/datalayer.feature

```diff
@@ -26,10 +26,21 @@ Feature: ERC725 Profile features
     Given I query DC node locally for last imported data set id
     Then response hash should match last imported data set id
 
-  Scenario: Basic dataset integrity with its xml
+  Scenario: Basic dataset integrity with it's xml
     Given I setup 1 node
     And I start the node
     And I use 1st node as DC
     And DC imports "importers/xml_examples/Basic/01_Green_to_pink_shipment.xml" as GS1
     Then imported data is compliant with 01_Green_to_pink_shipment.xml file
 
+  Scenario: Dataset immutability I
+    Given I setup 1 node
+    And I start the node
+    And I use 1st node as DC
+    And DC imports "importers/xml_examples/Basic/01_Green_to_pink_shipment.xml" as GS1
+    Given DC initiates the replication for last imported dataset
+    And I wait for 10 seconds
```

Comment:
Why waiting exact 10 seconds?

Reply:
Same question for all 10 seconds sentences.

Reply:
Ideally we could wait until replication window is closed, or similar, but I just want to wait until fingerprint is written on blockchain, so am sure it can be read later on.

Reply:
But I see waht you are after, I can refactor to wait for "offer written to blockchain" event